crowbar_framework/app/models/service_object.rb

```diff
@@ -452,7 +462,8 @@ def process_queue
           end
           next if queue_me
 
-          nodes_map = elements_to_nodes_to_roles_map(prop["deployment"][item["barclamp"]]["elements"])
+          nodes_map = elements_to_nodes_to_roles_map(prop["deployment"][item["barclamp"]]["elements"],
+                                                     prop["deployment"][item["barclamp"]]["element_order"])
```

Comment:
Metrics/LineLength: Line is too long. [107/80]<br>Style/AlignParameters: Align the parameters of a method call if they span more than one line.